EnhancedMediaPlayer/MediaPlayerView.swift

```diff
@@ -37,18 +36,30 @@ struct MediaPlayerView: View {
     }
 
     @ViewBuilder private func renderActionButtons() -> some View {
-        HStack {
-            renderRewind()
-                .frame(maxWidth: .infinity, alignment: .leading)
-            if viewModel.playerState == .finished {
-                renderReplay()
-            } else {
-                renderPlayPause()
+        ZStack {
+            HStack {
+                renderRewind()
+                    .frame(maxWidth: .infinity, alignment: .leading)
+                if viewModel.playerState == .finished {
+                    renderReplay()
+                } else {
+                    renderPlayPause()
+                }
+                renderForward()
+                    .frame(maxWidth: .infinity, alignment: .trailing)
             }
-            renderForward()
-                .frame(maxWidth: .infinity, alignment: .trailing)
+            .frame(maxWidth: .infinity)
+
+            if viewModel.showSettingsMenu {
+               SettingsMenu()
+                .padding(.bottom, 40)
+                .frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .bottomTrailing)
+            }
```

Comment:
You could insert this in a `VStack` so this can be ontop of the "bar" with the buttons instead

Reply:
Tried implementing the change, but it pushes the other buttons up:

https://github.com/profusion/ios-enhanced-media-player/assets/28735698/a35ad16d-5bb3-45a3-8822-76069291549a